media_types/mod.ts

```diff
@@ -0,0 +1,407 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+// This module is browser compatible.
+
+/** Provides utility functions for media types.
+ *
+ * This API is inspired by the GoLang [`mime`](https://pkg.go.dev/mime) package
+ * and [jshttp/mime-types](https://github.com/jshttp/mime-types).
+ *
+ * @module
+ */
+
+import db from "https://raw.githubusercontent.com/jshttp/mime-db/v1.52.0/db.json" assert {
```

Comment:
I think we should copy this file into std, and should avoid the 3rd party dependency

Reply:
Done...